### PR TITLE
Code Refactoring & Fixes

### PR DESCRIPTION
**Pull request by mwzheng**
- Refactoring of DataSection & ResultSection
- Fixed broken Download in some tabs
- Fixed broken Random Forest tabs

### CODE REVIEW

File: src/components/Analyze/DifferentiaExpAnalysisResults/D3Graphics/FoldChangeAnalysis/FoldChangeAnalysis.jsx
Comment (andrewlapointe) [inline src/components/Analyze/DifferentiaExpAnalysisResults/D3Graphics/FoldChangeAnalysis/FoldChangeAnalysis.jsx:1]: All good here

**Review by andrewlapointe — APPROVED** [review]
All my files look good to go.

**Review by Shubhs0411 — APPROVED** [review]
Looks good to me. Thank you Mickey.

**Review by AnthonyNVo — APPROVED** [review]
Looks good to me

**Pull request merged**